7zip.yaml

```diff
@@ -53,6 +53,5 @@ subpackages:
 
 update:
   enabled: true
-  manual: true
```

Comment:
I think this one needs to stay, but really we could just change the version to remove the separator.

Reply:
> I think this one needs to stay, but really we could just change the version to remove the separator.

Should I implement that now?

Reply:
If you want to.  2301 > 23.01 so should be fine.

Reply:
Done.